drafts/draft-0002.md

```diff
@@ -0,0 +1,121 @@
+5. `network_passphrase` (optional) - Only need to set if this transaction is for a network other than the public network.
+
+**Example 1 - Payment Operation with source account needing replacement**:
+
+`stellar://tx/?xdr=AAAAAL6Qe0ushP7lzogR2y3vyb8LKiorvD1U2KIlfs1wRBliAAAAZAAAAAAAAAAAAAAAAAAAAAAAAAABAAAAAAAAAAEAAAAABEz4bSpWmsmrXcIVAkY2hM3VdeCBJse56M18LaGzHQUAAAAAAAAAAACadvgAAAAAAAAAAA`
+
+**Example 2 - Change Trust Operation with submit endpoint specified**:
+
+`stellar://tx/?xdr=AAAAAP%2Byw%2BZEuNg533pUmwlYxfrq6%2FBoMJqiJ8vuQhf6rHWmAAAAZAB8NHAAAAABAAAAAAAAAAAAAAABAAAAAAAAAAEAAAAA%2F7LD5kS42DnfelSbCVjF%2Burr8GgwmqIny%2B5CF%2FqsdaYAAAAAAAAAAACYloAAAAAAAAAAAA&submit=url%3Ahttps%3A%2F%2FsomeSigningService.com%2Fa8f7asdfkjha&pubkey=GAU2ZSYYEYO5S5ZQSMMUENJ2TANY4FPXYGGIMU6GMGKTNVDG5QYFW6JS&msg=order%20number%2024`
+
+### Operation `pay`
+The `pay` operation represents a request to pay a specific address with a specific asset, regardless of the source asset used by the payer. If the payer decides to use a different source asset then the wallet should leverage the [path payment operation](https://www.stellar.org/developers/guides/concepts/list-of-operations.html#path-payment) to achieve this. The parameters for the `pay` operation are as follows:
+1. `destination` (required) - A valid account ID or payment address
+2. `amount` (required) - Amount that destination will receive
+3. `asset_code` (optional) - Asset code (XLM if not present) destination will receive
+4. `asset_issuer` (optional) - Account ID of asset issuer (XLM if not present) destination will receive
+5. `memo` (optional) - Can be a memo to be included in the payment / path payment. Memos of type `MEMO_HASH` and `MEMO_RETURN` should be base64 encoded and the URL encoded. Memos of type `MEMO_TEXT` should be URL-encoded.
+6. `memo_type` (optional) - One of `MEMO_TEXT`, `MEMO_ID`,`MEMO_HASH`, `MEMO_RETURN`. See [transaction guide](https://www.stellar.org/developers/guides/concepts/transactions.html#memo) for a description of these values. 
+7. `submit` (optional) - If this value is omitted then the URI handler should create and submit the proper transaction to the network. If the value is present then it should be interpreted as a URL-encoded callback. The URL-encoded callback will be prefixed with its own namespace to denote whether this is a `url` callback or some other form of callback. In the case where it is a url callback (denoted by `url:`) the URI handler should send the signed XDR to this url in a `POST` request with `Content-Type` set to `application/x-www-form-urlencoded` with the data fields `xdr` containing the signed XDR (URL-encoded). If there are any query params specified in the URL callback then those should be included in the URL when submitting. For now only `url` callback types are supported.
```

Comment:
I'm concerned about only allowing the `url` callback type. We need to have a good story for how a client side app communicates with the signer app without setting up a backend. Specifically, I'm thinking about single page apps communicating with browser extensions (cryptokitties + metamask style).

One way to do so is to have the single page app register a [custom protocol handler](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/registerProtocolHandler) and have the submission callback refer to that. That can work but will create an awkward user experience (the browser will ask the user if it approves this webpage to handle "Stellar Response" links or something like that)

Usually, webpage <-> extension communication happen through `postMessage()`, but that requires the website to know of a specific extension in advance ([example](https://gist.github.com/johnschimmel/fad562a4352e93353a91)) rather than being able to communicate with any extension that supports the protocol.

I wonder if I'm missing another option here. @nullstyle @bartekn ?